tools/pd-ctl/pdctl/command/location_command.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+
+	"github.com/spf13/cobra"
+)
+
+// NewLocationCommand return a member subcommand of rootCmd
```

Comment:
I think this feature is also could be used for user.

Reply:
Current implementation is only for test usage.

As described in #3396 : dc location is not persisted and only built from config, so it will be inconsistent after a reboot of pd server which dc location updated before.
Issue seems that only test usage needs this feature so I decided not to add a persist operation. But it can be done if need.

Reply:
Initially, it was only planned to support test use, but for now seems like @Yisaer said, it could be used by the users. And the configuration of `dc-location` and Local TSO will be changed recently, you can take a look at #3414. We can have a further discussion there.